src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { useState } from "react";
+import * as UI from "./style";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import moment from "moment";
+
+export interface SocialInfo {
+  type: string;
+  link: string;
+}
+
+export interface InfoProps {
+  title: string;
+  description: string;
+  startTime: Date;
+  endTime: Date;
+  speakerName: string;
+  speakerAbout: string;
+  speakerSocials: SocialInfo[];
+  relatedActivities?: string[];
+}
+
+export const formatTime = (time: Date, endTime: Date): string => {
+  /**
+   * returns string in Day HH:MM(AM/PM) format
+   * @param time a date object
+   */
+  return (
+    moment(time).format("dddd h:mma") + " to " + moment(endTime).format("h:mma")
+  );
+};
+
+export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
+  const [interestedBtnText, setInterestedBtnText] = useState(
+    "🤔 Mark me interested"
+  );
+
+  const handleClick = () => {
+    if (interestedBtnText == "🙄 I'm no longer interested") {
+      setInterestedBtnText("🤔 Mark me interested");
+      console.log("user is interested");
+    } else {
+      setInterestedBtnText("🙄 I'm no longer interested");
+      console.log("user is no longer interested");
+    }
```

Comment:
These might be better in an enum so that we can control the different modes of the button

Reply:
i didn't use enum but I believe I did something simpler, could you check again and let me know if that works well?